test/runtests.jl

```diff
@@ -345,6 +382,10 @@ end
         callsite = only(callsites)
         info = callsite.info
         @test isa(info, Cthulhu.InvokeCallInfo)
+        if Cthulhu.EFFECTS_ENABLED
+            effects = Cthulhu.get_effects(info)
+            @test_broken Core.Compiler.is_total(effects) # don't have effects from InvokeCallInfo yet
```

Comment:
I think this may require #281 but not sure. The comment is wrong though, we do grab effects from `InvokeCallInfo`